backend/db/schema.rb

```diff
@@ -28,6 +27,23 @@
     t.index ["user_id"], name: "index_blacklisted_tokens_on_user_id"
   end
 
+
+  create_table "courses", force: :cascade do |t|
+    t.string "name"
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_courses_on_user_id"
+  end
```

Comment:
This is not related to your PR, right?

Reply:
yep. i just merged it, because i had conflict with it after pulling.